SpanishFilter/sections/general_extensions.txt

```diff
@@ -2,6 +2,8 @@
 !------- JS rules ---------------------!
 !--------------------------------------!
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/69974
+tuhentaionline.com#%#//scriptlet('abort-current-inline-script', 'document.getElementById', 'banner_deslizante')

```

Comment:
I guess cosmetic rule is enough here. Why do you add the scriptlet?

Reply:
Just to eliminate the cause, but sure is redundant. I'll remove this.

Reply:
Ok, I understand it now. For such cases we use a simple way to solve the issue, if it's possible.